framework/wazuh/core/tests/data/rules/0015-wazuh_rules.xml

```diff
@@ -88,7 +249,7 @@
     <if_sid>510</if_sid>
     <match>^Windows Audit</match>
     <description>Windows Audit event.</description>
-    <group>rootcheck,</group>
+    <group>pci_dss_10.6.1,hipaa_164.312.b,rootcheck,</group>
```

Comment:
These changes aren't part of this implementation. This file differs a little from **wazuh** and **wazuh-ruleset** repositories.
Is it correct to modify it?

Reply:
This was reverted in order to avoid changing the rules in the core repository.